modules/navier_stokes/doc/content/source/functormaterials/LinearFrictionFactorFunctorMaterial.md

```diff
@@ -0,0 +1,24 @@
+# LinearFrictionFactorFunctorMaterial
+
+!syntax description /FunctorMaterials/LinearFrictionFactorFunctorMaterial
+
```

Comment:
I would probably just remove these lines as it introduces the equation without formatting whereas you have it with formatting later in this file